pkg/cmd/install.go

```diff
@@ -129,7 +131,19 @@ func loadPublickey(path string) ssh.AuthMethod {
 	}
 	signer, err := ssh.ParsePrivateKey(key)
 	if err != nil {
-		panic(err)
+		if err.Error() != "ssh: cannot decode encrypted private keys" {
+			panic(err)
```

Comment:
Is there a way to do this without a panic? I think we could just return an error?

Reply:
Sure! We then get errors like this:
```
Enter passphrase for '/home/rkaufmann/.ssh/id_rsa': 
Error: unable to load the ssh key with path "/home/rkaufmann/.ssh/id_rsa": x509: decryption password incorrect
```

Reply:
I don't follow?

My suggestion is to return the error instead of using `panic`. Can you elaborate on why you would prefer to use `panic`?

Reply:
I already changed the code so that it returnes the error.
I'm gonna update the PR this afternoon.

> Error: unable to load the ssh key with path "/home/rkaufmann/.ssh/id_rsa": x509: decryption password incorrect

This was just a testrun with the error instead of a panic.

Reply:
No problems. It looked like my original code had a panic in anyway :laughing:

Reply:
I'd really like a quick unit test to cover the error checking too.

I have a simple guide here, which may help unless you're already used to writing them.

https://blog.alexellis.io/golang-writing-unit-tests/